domains/certificates/Query.API/API/Query.API/Projections/CertificatesByOwnerProjection.cs

```diff
@@ -1,119 +1,186 @@
+                         productionCertificateTransferred.CertificateId,
+                         $"Cannot transfer from {productionCertificateTransferred.Source}. {productionCertificateTransferred.CertificateId} cannot be found"
+                     );
+        var cert = source.Certificates[productionCertificateTransferred.CertificateId];
 
-    public Dictionary<Guid, CertificateView> Certificates { get; set; } = new();
+        RemoveCertificateFromSource(source, productionCertificateTransferred);
+        AddCertificateToTarget(operations, productionCertificateTransferred, cert);
+    }
 
-    public CertificateList ToApiModel()
+    private static void AddCertificateToTarget(IDocumentOperations operations,
+        ProductionCertificateTransferred productionCertificateTransferred, CertificateView certificateView)
     {
-        var certificates = Certificates.Values
-            .Select(c => new Certificate
+        var view = operations.Load<CertificatesByOwnerView>(productionCertificateTransferred.Target);
+
+        if (view == null)
+        {
+            view = new CertificatesByOwnerView
             {
-                Id = c.CertificateId,
-                GSRN = c.GSRN,
-                GridArea = c.GridArea,
-                DateFrom = c.DateFrom,
-                DateTo = c.DateTo,
-                Quantity = c.Quantity,
-                TechCode = c.TechCode,
-                FuelCode = c.FuelCode
-            });
-
-        return new CertificateList
+                Owner = productionCertificateTransferred.Target,
+                Certificates =
+                {
+                    [certificateView.CertificateId] = certificateView
+                }
+            };
+        }
+        else
         {
-            Result = certificates
-                .OrderByDescending(c => c.DateFrom)
-                .ThenBy(c => c.GSRN)
-                .ToArray()
-        };
+            view.Certificates.Add(certificateView.CertificateId, certificateView);
+        }
+
+        views.Add(view);
     }
-}
 
-public class CertificateView
-{
-    public Guid CertificateId { get; set; }
-    public long DateFrom { get; set; }
-    public long DateTo { get; set; }
-    public long Quantity { get; set; }
-    public string GSRN { get; set; } = "";
-    public string GridArea { get; set; } = "";
-    public string TechCode { get; set; } = "";
-    public string FuelCode { get; set; } = "";
-    public CertificateStatus Status { get; set; }
-}
+    private static void RemoveCertificateFromSource(CertificatesByOwnerView source,
+        ProductionCertificateTransferred productionCertificateTransferred)
+    {
+        source.Certificates.Remove(productionCertificateTransferred.CertificateId);
+        views.Add(source);
+    }
 
-public enum CertificateStatus
-{
-    Creating = 1,
-    Issued = 2,
-    Rejected = 3
-};
+    private static CertificatesByOwnerView GetCertificatesByOwnerView(IDocumentOperations operations,
```

Comment:
According to SonarCloud there are no tests that cover this method. Why is that the case?